src/common.jl

```diff
@@ -225,6 +225,10 @@ function _pdf(d::Distribution{ArrayLikeVariate{N}}, x::AbstractArray{<:Real,N})
     return exp(@inbounds logpdf(d, x))
 end
 
+function pdf(d::Distribution{ArrayLikeVariate{N}}, x::Missing) where N
+    return missing
+end
```

Comment:
A bit shorter and also with a definition for `logpdf`:
```suggestion
pdf(::Distribution{<:ArrayLikeVariate}, ::Missing) = missing
logpdf(::Distribution{<:ArrayLikeVariate}, ::Missing) = missing
```